Cargo.toml

```diff
@@ -7,6 +7,9 @@ members = [
     "packages/utils/crates/utils-metrics"
 ]
 
+[workspace.dependencies]
+wasm-pack = "0.10.3"
```

Comment:
I recommend to use prebuilt binaries since compiling wasm-pack and wasm-opt takes 10 minutes - for each utility.

Reply:
let's do this as an optimisation afterwards